src/components/new-request/business-lookup.vue

```diff
@@ -5,13 +5,13 @@
     id="business-lookup"
   >
     <v-autocomplete
+      v-model:search-input="searchField"
```

Comment:
No, don't change this as it doesn't work in Vue2/Vuetify 2.

I think a lot of these changes were made automatically when you ran lint locally (without the no-fix option). Please revert all the changes except YOURs.

Reply:
Got it